tools/browsers-lists/browsers-supported.json

```diff
@@ -2,107 +2,117 @@
-      "platformName": "Windows 11",
+      "platformName": "Windows 10",
+      "platform": "Windows 10",
       "version": "110",
       "browserVersion": "110"
     },
     {
       "browserName": "chrome",
-      "platform": "Windows 11",
       "platformName": "Windows 11",
-      "version": "106",
-      "browserVersion": "106"
-    },
-    {
-      "browserName": "chrome",
       "platform": "Windows 11",
-      "platformName": "Windows 11",
-      "version": "103",
-      "browserVersion": "103"
+      "version": "113",
+      "browserVersion": "113"
     }
   ],
   "edge": [
     {
       "browserName": "MicrosoftEdge",
-      "platform": "Windows 10",
       "platformName": "Windows 10",
-      "version": "112",
-      "browserVersion": "112"
-    },
-    {
-      "browserName": "MicrosoftEdge",
       "platform": "Windows 10",
-      "platformName": "Windows 10",
-      "version": "109",
-      "browserVersion": "109"
+      "version": "103",
+      "browserVersion": "103"
     },
     {
       "browserName": "MicrosoftEdge",
-      "platform": "Windows 10",
-      "platformName": "Windows 10",
+      "platformName": "Windows 11",
+      "platform": "Windows 11",
       "version": "106",
       "browserVersion": "106"
     },
     {
       "browserName": "MicrosoftEdge",
-      "platform": "Windows 11",
       "platformName": "Windows 11",
-      "version": "103",
-      "browserVersion": "103"
-    }
-  ],
-  "safari": [
-    {
-      "browserName": "safari",
-      "platform": "Mac 13",
-      "version": "16",
-      "acceptInsecureCerts": false
+      "platform": "Windows 11",
+      "version": "109",
+      "browserVersion": "109"
     },
     {
-      "browserName": "safari",
-      "platform": "Mac 12",
-      "version": "15",
-      "acceptInsecureCerts": false
+      "browserName": "MicrosoftEdge",
+      "platformName": "Windows 11",
+      "platform": "Windows 11",
+      "version": "112",
+      "browserVersion": "112"
     }
   ],
   "firefox": [
     {
       "browserName": "firefox",
+      "platformName": "Windows 10",
       "platform": "Windows 10",
-      "version": "112"
+      "version": "103",
+      "browserVersion": "103"
     },
     {
       "browserName": "firefox",
+      "platformName": "Windows 10",
       "platform": "Windows 10",
-      "version": "109"
+      "version": "106",
+      "browserVersion": "106"
     },
     {
       "browserName": "firefox",
+      "platformName": "Windows 10",
       "platform": "Windows 10",
-      "version": "106"
+      "version": "109",
+      "browserVersion": "109"
     },
     {
       "browserName": "firefox",
+      "platformName": "Windows 10",
       "platform": "Windows 10",
-      "version": "103"
+      "version": "112",
+      "browserVersion": "112"
+    }
+  ],
+  "safari": [
+    {
+      "browserName": "safari",
+      "platformName": "Mac 12",
+      "platform": "Mac 12",
+      "version": "15",
+      "browserVersion": "15"
```

Comment:
Also missing insecureCerts property in safari desktop

Reply:
fixed, thanks for catching